test/serializers/default_include_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
```

Comment:
oh, sorry. I guess I wrote too quickly. Here is there scenario I'd like tested:
```
ParentSerializer
  has_many Child

ChildSerializer
  default_include [:grand_child]
```

when serializing the parent with `include: 'children'` (this was probably a bad example, thanks English pluralization), only the child should show up, and not the grand child.